Create release process (#45)
kyma-project/helm-broker: README.md

make integration-test
```

> ### Update the chart's images tag

Reviewer: ```suggestion
### Update chart's images tag
```

> Change the chart's tags version using the command:

Reviewer: ```suggestion
To change the chart's tags version, run this command:
```

```bash
make VERSION=v0.0.1 DIR=/pr tag-chart-images
```

> Above command overrides the images tag in the `charts/helm-broker/values.yaml` file to:

Reviewer: ```suggestion
This command overrides the images tag in the `charts/helm-broker/values.yaml` file to:
```
